src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -2159,5 +2161,163 @@ public unsafe void M2()
                 //             await Task.Delay(2);
                 Diagnostic(ErrorCode.ERR_AwaitInUnsafeContext, "await Task.Delay(2)").WithLocation(33, 13));
         }
+
+        [Fact, WorkItem(16167, "https://github.com/dotnet/roslyn/issues/16167")]
+        public void Bug16167()
```

Comment:
It doesn't look like the PR adds any success tests. I think we should know if scenario like this is going to work correctly:
```
const int x =1;
void Local(int p = x) 
{
    ...
}
```